### PR TITLE
LPS-156347 | master

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LPS-156347
Submit to master.

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -720,6 +720,57 @@ definition {
...
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Action Name",
+				active = "true",
+				inputExpression = "customObjectField == 'Entry Test'",
+				thenAction = "Webhook",
+				urlAction = "http://localhost:8080",
+				whenAction = "On After Add");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:757]: Hey @chaowang-123, would this on CI? We may need to add the property to skip portal instances for this test. Can you verify?

If it works as is, you can forward this to BChan.

**Review by timpak — COMMENTED** [review]
@chaowang-123 Can you check if this test will pass on CI? We may need to skip portal instances for this to work

**Comment by chaowang-123** [comment]
Hi @timpak 
resend [here](https://github.com/timpak/liferay-portal/pull/352)

**Pull request closed without merging**